### DISCUSSION TITLE
Update 2.x removeAllListeners documentation

### DISCUSSION THREAD

**Discussion by rcuching** [mswjs/msw#2158] [General]
### Scope

Improves an existing behavior

### Compatibility

- [ ] This is a breaking change

### Feature description

While debugging msw 2.2.14 I was able to find `server.events.removeAllListeners`. I was able to find the 1.0 docs related to this feature https://v1.mswjs.io/docs/extensions/life-cycle-events#removealllisteners. Requesting documentation update to reflect current features unless its planned to be deprecated and just not annotated as such.

**Comment by kettanaito** [comment]
Hi, @rcuching. Thanks for pointing this out! Looks like those paragraphs got lost in the v2 migration. 

I've updated the Life-cycle events API page in the v2 documentation to mention the `.removeListener()` and `.removeAllListeners()` methods:

- [Life-cycle events API, Removing listeners](https://mswjs.io/docs/api/life-cycle-events#removing-listeners)

Please let me know if you discover any other pieces missing in the documentation (that haven't been deprecated by v2)! Thanks.

**Reply by rcuching** [reply]
awesome, thank you!
